web_app/contract_tools/mixins/alert.py

```diff
@@ -0,0 +1,54 @@
+"""
+This module contains the alert mixin class.
+"""
+from typing import List
+from web_app.db.crud import UserDBConnector
+from web_app.contract_tools.mixins.dashboard import DashboardMixin
+from enum import Enum
+
+class Status(Enum):
```

Comment:
why did you created class if we have it already
https://github.com/djeck1432/spotnet/blob/main/web_app/db/models.py#L25